crates/sui-sdk-crypto/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "sui-sdk-crypto"
+version = "0.0.0"
+authors = ["Brandon Williams <brandon@mystenlabs.com>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[package.metadata.docs.rs]
+# To build locally:
+# RUSTDOCFLAGS="--cfg=doc_cfg -Zunstable-options --generate-link-to-definition" RUSTC_BOOTSTRAP=1 cargo doc --all-features --no-deps --open
+all-features = true
+rustdoc-args = [
+    # Enable doc_cfg showing the required features.
+    "--cfg=doc_cfg",
+
+    # Generate links to definition in rustdoc source code pages
+    # https://github.com/rust-lang/rust/pull/84176
+    "-Zunstable-options", "--generate-link-to-definition"
+]
+
+[features]
+default = []
+ed25519 = ["dep:ed25519-dalek", "dep:rand_core", "sui-sdk/hash", "sui-sdk/serde"]
+secp256r1 = ["dep:p256", "dep:rand_core", "sui-sdk/hash", "sui-sdk/serde"]
+secp256k1 = ["dep:k256", "dep:rand_core", "signature/std", "sui-sdk/hash", "sui-sdk/serde"]
+
+[dependencies]
+signature = "2.2"
+sui-sdk = { version = "0.0.0", path = "../sui-sdk", default-features = false }
+
+# RNG support
+rand_core = { version = "0.6.4", optional = true }
+
+# ed25519 support
+ed25519-dalek = { version = "2.1.1", optional = true }
+
+# secp256r1 support
+p256 = { version = "0.13.2", default-features = false, features = ["ecdsa", "std"], optional = true }
+
+# secp256k1 support
+k256 = { version = "0.13.4", default-features = false, features = ["ecdsa"], optional = true }
```

Comment:
i believe similar story here too (fastcrypto uses a different lib)

- https://github.com/MystenLabs/fastcrypto/blob/main/fastcrypto/src/secp256k1/mod.rs#L36 
- https://github.com/rust-bitcoin/rust-secp256k1/

Reply:
this library does have an audit though: https://www.nccgroup.com/us/research-blog/public-report-entropyrust-cryptography-review/